person.rb

```diff
@@ -0,0 +1,42 @@
+class Person
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = 0
```

Comment:
- [ ] Let's make use of random number to generate `id` each time a new `Person` is created. This will make each new Person have a unique `id`. you can replace line `3` with `@id = Random.rand(1..1000)`